anyway/parsers/registered.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+import csv
+import glob
+import io
+import logging
+import os
+import re
+import sys
+from datetime import datetime
+
+import six
+from flask.ext.sqlalchemy import SQLAlchemy
+
+from ..models import RegisteredVehicle, City
+from ..utilities import init_flask, time_delta
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+# Headless servers cannot use GUI file dialog and require raw user input
+fileDialog = True
+try:
+    import tkFileDialog
+except (ValueError, ImportError):
+    fileDialog = False
+
+app = init_flask()
+db = SQLAlchemy(app)
+
+_manual_assing_on_city_name = {
+    'הרצלייה': 'הרצליה',
+    'תל אביב-יפו': 'תל אביב -יפו',
+    'אפרתה': 'אפרת',
+    "באקה-ג'ת": 'באקה אל-גרביה',
+    'בוקעאתה': 'בוקעאתא',
+    "ג'ש (גוש חלב)": "ג'ש )גוש חלב(",
+    'דבורייה': 'דבוריה',
+    'טובא-זנגרייה': 'טובא-זנגריה',
+    'יהוד': 'יהוד-מונוסון',
+    'יהוד-נווה אפרים': 'יהוד-מונוסון',
+    "כאוכב אבו אל- היג'א": "כאוכב אבו אל-היג'א",
+    "כעביה-טבאש- חג'אג'רה": "כעביה-טבאש-חג'אג'רה",
+    'מודיעין-מכבים- רעות' : 'מודיעין-מכבים-רעות',
+    'נהרייה' : 'נהריה',
+    "פקיעין (בוקייעה)" : "פקיעין )בוקייעה(",
+    'פרדסייה' : 'פרדסיה',
+    'קציר-חריש' : 'קציר',
+    'שבלי-אום אל-גנם' : 'שבלי - אום אל-גנם',
+}
+
+
+class DatastoreImporter(object):
+    _header_size = 12
+    _in_encode = "utf-8"
+    _report_year = 0
+    _population_year = 0
```

Comment:
Fields should be set in `__init__` rather than as class variables.

Reply:
will do